src/runner/patch-round.js

```diff
@@ -0,0 +1,110 @@
+import { BIG_DECIMAL, DECIMAL_128 } from "../constants.js";
+
```

Comment:
The nice part about using the module type is that we can make this its own file, since it has so much logic. I expect some other patches will be the same, so we can keep the lengths short and organized.